contracts/dao/Getters.sol

```diff
@@ -146,6 +177,85 @@ contract Getters is State {
         return _state.accounts[owner].couponAllowances[spender];
     }
 
+    // DIP-10
+    function balanceOfCDSDBonded(address account) public view returns (uint256) {
+        uint256 amount = depositedCDSDByAccount(account).mul(_state10.globalInterestMultiplier).div(intrestMultiplierEntryByAccount(account));
+
+        uint256 cappedAmount = cDSDBondedCap(account);
+
+        return amount > cappedAmount ? cappedAmount : amount;
+    }
+
+    function cDSDBondedCap(address account) public view returns (uint256) {
+        return depositedCDSDByAccount(account).add(earnableCDSDByAccount(account)).sub(earnedCDSDByAccount(account));
+    }
+
+    function depositedCDSDByAccount(address account) public view returns (uint256) {
+        return _state10.accounts[account].depositedCDSD;
+    }
+
+    function intrestMultiplierEntryByAccount(address account) public view returns (uint256) {
+        return _state10.accounts[account].depositedCDSD;
+    }
+
+    function earnableCDSDByAccount(address account) public view returns (uint256) {
+        return _state10.accounts[account].earnableCDSD;
+    }
+
+    function earnedCDSDByAccount(address account) public view returns (uint256) {
+        return _state10.accounts[account].earnedCDSD;
+    }
+
+    function redeemedCDSDByAccount(address account) public view returns (uint256) {
+        return _state10.accounts[account].redeemedCDSD;
+    }
+
+    function getRedeemedThisExpansion(address account) public view returns (uint256) {
+        uint256 currentExpansion = _state10.expansionStartEpoch;
+        uint256 accountExpansion = _state10.accounts[account].lastRedeemedExpansionStart;
+
+        if (currentExpansion != accountExpansion) {
+            return 0;
+        }else{
+            return _state10.accounts[account].redeemedThisExpansion;
+        }
+    }
+
+    function getCurrentRedeemableCDSDByAccount(address account) public view returns (uint256) {
+        return dip10TotalRedeemable()
+            .mul(balanceOfCDSDBonded(account))
+            .div(totalCDSDBonded())
+            .sub(getRedeemedThisExpansion(account));
+    }
+
+
+
+
+    function totalCDSDDeposited() public view returns (uint256) {
+        return _state10.totalCDSDDeposited;
+    }
+
+    function totalCDSDEarnable() public view returns (uint256) {
+        return _state10.totalCDSDEarnable;
+    }
+
+    function totalCDSDEarned() public view returns (uint256) {
+        return _state10.totalCDSDRedeemed;
```

Comment:
`_state10.totalCDSDEarned`?

Reply:
May have been looked over. Thanks for pointing it out.